com/epam/brigada_5/java/lesson_1/task_1/Array.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.brigada_5.java.lesson_1.task_1;
+
+import java.util.Arrays;
+
+public class Array {
+
+    int[] array = new int[20];
+    int avg;
+    int min;
+
+    public void arrayFilling() {
+        for (int i = 0; i < 20; i++) {
+            array[i] = (int) (Math.random() * 21) - 10;
+            System.out.println("Element №" + (i+1) + " = " + array[i]);
+        }
+    }
+
+    public void averageOfArray() {
+        int sum = 0;
+
+        for (int i = 0; i < 20; i++) {
+            sum = sum + array[i];
+        }
+        avg = sum / 20;
+        System.out.println("Average of array is " + avg);
+    }
+
+    public void minOfArray() {
+        Arrays.sort(array);
+        min = array[0];
+        System.out.println("Minimum of array is " + min);
+    }
+
+    public void difference() {
+        int diff;
+        diff = avg - min;

```

Comment:
нет смысла в этом разделении
```suggestion
        --int diff;
        int diff = avg - min;
```